swaglyrics/cli.py

```diff
@@ -13,7 +13,7 @@ def clear() -> None:
 	os.system('cls' if os.name == 'nt' else 'clear')  # clear command window
 
 
-brc = re.compile(r'([(\[](feat|ft)[^)\]]*[)\]]|- .*)', re.I)  # matches braces with feat included or text after -
+brc = re.compile(r'([(\[](feat|ft|From)[^)\]]*[)\]]|- .*)', re.I)  # matches braces with feat included or text after -
```

Comment:
This technically works but it is a bit too generalized, we know the from in the song is something like (From "<movie name>") so we should check for the quotes too, to avoid possible false positives.

Reply:
ya, you are right, but when we try to check quotes too. It became a little bit of a complex solution.

Reply:
yeah that's okay